Assets/Project/Scripts/Modules/StageSelectScene/StageSelectDirector.cs

```diff
@@ -176,8 +181,35 @@ private void SetupCoachMark()
         private async UniTask CheckReleaseAnimationAsync()
         {
             await CheckClearCrystalStageAsync();
-            CheckReleaseNewStageAsync().Forget();
-            UpdateProgressBar();
+            await CheckReleaseNewStageAsync();
+            await UpdateProgressBar();
```

Comment:
枝の解放を待ってから遷移するようにすると、
「結晶じゃないステージ」 && 「枝の解放があるステージ」の時に、枝の解放演出中にプログレスバーが最新の状態にならないようになってしまった。

`UpdateProgressBar`は`await`する処理が入らない(`**Asynk`という命名にしてもしていない)ので、以下の形に戻したい。 + 枝の解放アニメーションと処理順序を入れ替えたい。
```suggestion
            UpdateProgressBar();
            await CheckReleaseNewStageAsync();
```

Reply:
[5a4e5ee](https://github.com/LITO-apps/Treevel-client/pull/1511/commits/5a4e5ee38bf7bfdaae9d09586b82792ff3b1bc69)
`TreeProgressBarController.UpdateProgressBar`の`isAnimate`が`true`と`false`で同期関数か非同期関数かで変わるのはわかりづらかった・・・